classifiers/nlp/ZeroShotTFClassifier/README.md

```diff
@@ -0,0 +1,4 @@
+# ZeroShotTFClassifier
```

Comment:
Hey @julianpetrich ,

We would need some better and more insightful Readme for this specific executor, with examples on how to use and a little more explanation of the parameters and the inner logic.

You can see how these changes are being applied to the rest of executors